Makefile

```diff
@@ -40,6 +40,6 @@ start:
 build: check-env clone
 	@ printf "\n[${GREEN} Generating environment variables... ${NO_COLOR}]\n"
 	cd ${SDX_HOME}/sdx-common && pip install .; cd -
-	cd ${SDX_HOME}/sdx-ops && ${PYTHON3} sdx/ops/configure.py --env > ${SDX_HOME}/sdx-compose/env/private.env ;cd -
+	cd ${SDX_HOME}/sdx-ops && python sdx/ops/configure.py --env > ${SDX_HOME}/sdx-compose/env/private.env ;cd -
```

Comment:
Why changing from the parameterised call? The local installed python3 may not be available as a specific `python` command in the path